src/i965_drv_video.c

```diff
@@ -892,7 +892,9 @@ i965_validate_config(VADriverContextP ctx, VAProfile profile,
         } else if (!HAS_VP9_DECODING_PROFILE(i965, profile) &&
                    !HAS_VP9_ENCODING(i965) &&
                    !HAS_LP_VP9_ENCODING(i965) &&
-                   !i965->wrapper_pdrvctx) {
+                   (!i965->wrapper_pdrvctx ||
+                    /* Hybrid driver does not support VP9.2 */
+                    i965->wrapper_pdrvctx && profile == VAProfileVP9Profile2)) {
```

Comment:
How about to call  vaQueryConfigEntrypoints to get the entrypoint list from the hybrid driver first, then check whether the specified entrypoint is included in the returned list  ? 

               CALL_VTABLE(pdrvctx, va_status,                                                                                                                                                
                            vaQueryConfigEntrypoints(pdrvctx, VAProfileVP9Profile2,
                                                     entrypoint_list,
                                                     num_entrypoints));

Reply:
Thanks for the feedback. I'll give it a try.

I initially went with a simplistic approach to maintain consistency with an earlier if else block at https://github.com/intel/intel-vaapi-driver/blob/master/src/i965_drv_video.c#L888-L891 . If the wrapper context is set, it always returns success for VP9 profile 0 decode.

Reply:
If you are willing, you may provide a patch to fix it for VP9 profile 0 too, thanks!

Reply:
Hi, Sorry about the delays. I have an updated patch but still need to test it on a few more devices. Thanks.